app/models/order.rb

```diff
@@ -27,6 +21,7 @@ def add_order_item item
     end
 
     order_items.each do |item|
+      puts "Quantity: #{item.inspect}, on hand count: #{item.source.on_hand_count}"
```

Comment:
this looks like a debugging log, should remove debug statements before merging in with master / production code